hazelcast/src/main/java/com/hazelcast/flakeidgen/impl/FlakeIdGeneratorService.java

```diff
@@ -54,5 +73,19 @@ public DistributedObject createDistributedObject(String name) {
 
     @Override
     public void destroyDistributedObject(String name) {
+        statsMap.remove(name);
+    }
+
+    @Override
+    public Map<String, LocalFlakeIdGeneratorStats> getStats() {
+        return new HashMap<String, LocalFlakeIdGeneratorStats>(statsMap);
+    }
+
+    public void incrementNewId(String name) {
```

Comment:
Please add javadoc to the new methods. Also, can you rename the method to reflect that it increments the statistics? This way I think it actually increments the ID generated by a flake ID instance.

Reply:
It counts number of NewIdBatch calls, the name should reflect that, such as `incNewIdBatchCounter`.